pom.xml

```diff
@@ -167,6 +167,11 @@
             <url>http://download.eclipse.org/releases/luna</url>
         </repository>
         <repository>
+            <id>testng-p2-repo</id>
+            <layout>p2</layout>
+            <url>http://dl.bintray.com/testng-team/testng-p2/</url>
```

Comment:
for now, we are using the beta update site, will switch to the release repo when testng-p2 public released.